src/core/stdcpp/new_.d

```diff
@@ -40,6 +40,39 @@ extern (C++, "std")
 }
 
 
+///
+T* cpp_new(T, Args...)(auto ref Args) if (!is(T == class))
```

Comment:
What about dynamic and associative arrays ?

Reply:
Yeah I was pondering that.
You can do `cpp_new!(T[N])`, which will allocate a static array of the desired length... maybe that's fine?
Alternatively we have some other form that receives a length... but I wasn't sure how to overload the function. Maybe it should be a separate function like `cpp_new_array()`?

Reply:
Yeah `cpp_new!(T[N])` is fine but won't `cpp_new!(T[])` allocate a `T[]*` technically ?

Reply:
Yes, `T[]` will allocate a `T[]*`, which seems... fine?
If we want it to allocate a dynamic array and return a slice, then we need to decide how to supply the length.
Overload (difficult because `Args...`)? Separate function with a different name?